### PR TITLE
Paste text instead of typing it in test

### PR DESCRIPTION
**Pull request by WULCAN**
The TagDialog tests `creates a basic tag` and `When creating a new group, sends the new group properties to the onSubmit callback instead of groupId` are consistently slow enough on my laptop to trip Jest's default timeout of 5 seconds.

If I run the tests serially, it more or less narrowly fits into 5 seconds but by default, Jest runs tests in parallel and then execution is always too slow for this particular test.

```
 FAIL  src/features/tags/components/TagManager/components/TagDialog/TagDialog.spec.tsx (24.71 s)
  ● <TagDialog /> › creates a basic tag

    thrown: "Exceeded timeout of 5000 ms for a test.
    Use jest.setTimeout(newTimeout) to increase the timeout value, if this is a long-running test."

      21 |   });
      22 |
    > 23 |   it('creates a basic tag', async () => {
         |   ^
      24 |     const { getByTestId } = render(
      25 |       <TagDialog
      26 |         groups={[]}

      at src/features/tags/components/TagManager/components/TagDialog/TagDialog.spec.tsx:23:3
      at Object.<anonymous> (src/features/tags/components/TagManager/components/TagDialog/TagDialog.spec.tsx:12:1)

  ● <TagDialog /> › 
      When creating a new group, sends the new group properties
      to the onSubmit callback instead of groupId
    

    thrown: "Exceeded timeout of 5000 ms for a test.
    Use jest.setTimeout(newTimeout) to increase the timeout value, if this is a long-running test."

      48 |   });
      49 |
    > 50 |   it(`
         |   ^
      51 |       When creating a new group, sends the new group properties
      52 |       to the onSubmit callback instead of groupId
      53 |     `, async () => {

      at src/features/tags/components/TagManager/components/TagDialog/TagDialog.spec.tsx:50:3
      at Object.<anonymous> (src/features/tags/components/TagManager/components/TagDialog/TagDialog.spec.tsx:12:1)
```

Profiling `When creating a new group, sends the new group properties` I found that a lot of time is spent typing the characters into the fields. While it's more realistic to type the text, _how_ the text is entered does not seem to be what the authors were concerned about.

## Description

This PR pastes instead of typing text into fields, cutting enough of the execution time to comfortably fit in Jest's default timeout, even when other tests are running concurrently.

With `paste` instead of `keyboard`, execution time is approximately halved on my potato machine. It won't make a significant impact on suite time, but it makes a big difference for me, in that the suite succeeds. =)

### main

`When creating a new group, sends the new group properties`
image

### fixed

`When creating a new group, sends the new group properties`
image

## Notes to reviewer

Profiling can be done with Node's `--cpu-prof`:

```
node --cpu-prof node_modules/jest-cli/bin/jest.js -i \
    --testPathPattern TagDialog
    --testNamePattern 'When creating a new group, sends the new group properties'
```

You can view recorded profiles as a flame graph with Visual Studio Code extension Flame Chart Visualizer for JavaScript Profiles (`ms-vscode.vscode-js-profile-flame`)

The other tests for `<TagDialog />` use keyboard too, but not with as many total characters, which makes them fast enough.

### CODE REVIEW

**Comment by richardolsson** [comment]
Wow that's very thorough work and an interesting find. I wonder if it's slow because the rate of (simulated) typing is slow, or because of poor performance in the UI. Do you experience any problems when you're using the UI on your machine?

**Review by richardolsson — APPROVED** [review]
I think this looks fine, and I will go ahead and merge, but I'm still curious about performance (see previous comment @WULCAN).

**Comment by WULCAN** [comment]
> Wow that's very thorough work and an interesting find. I wonder if it's slow because the rate of (simulated) typing is slow, or because of poor performance in the UI. Do you experience any problems when you're using the UI on your machine?

I thought it was just the test framework simulating a realistic time between key presses but that does not seem to be the case. By default it uses `delay` `0` which shouldn't bring any more delay than `setTimeout` with `0`. Looking a little more at the profiler it just seems to do a lot of work on each `keyboard` call.

Performance seems a little better when I test a similar scenario on `yarn devserver` but there's still a lot of work being done on each character input into the fields of the Create Tag dialog and I see at least one yank event over 140 ms.

It's probably more of a battery-drain problem than a latency-problem. Maybe a young user can detect and be annoyed by 100 ms delay per key. Battery-wise, I see 50--90 % CPU usage for ~100 ms which seems waste-full.

Profiling the Tag name field in Firefox, I see that a lot is rendered on each character input: `ButtonBase`, `Autocomplete`, `FormControl`, `TouchRipple`, `ModalUnstyled`, `Fade`, `TextField`, `InputBase`, `Dialog`, `Modal`, `InputLabel`, `FormLabel`, `Button`, `FormGroup` and `SwitchBase`. 

A large part of that is spent in `createStyled/</Styled<` which I suspect has something to do with Emotion but unfortunately, I have no source maps functioning in Firefox profiler either.

In conclusion, there seems to be a performance problem, at least in our Jest tests and in our devserver but it's probably not big enough to be detected by most users as a slow UI.

The next step is probably to profile a production build and then to profile a lot more scenarios than just inputting characters into this `TagDialog`. I don't think we have evidence that this part is our worst battery-drain in Zetkin.

@richardolsson
